website/docs/developer/research/research-lambda-container.md

```diff
@@ -0,0 +1,144 @@
+---
+id: research-lambda-container
+title: Using Lambda functions as container research Notes
+description: Research Notes on 
+---
+
+## Overview
+
+Lambda functions can now be deployed as container images using Docker instead of zip files. This research webpage discusses how lambda functions can be prototyped as container and its pros and cons.
+
+## Elastic Container Registry (ECR)
+
+AWS Elastic Container Registry(ECR) is used to store container images for lambdas and is fully managed by AWS. It hosts the images in a highly available and scalable architecture, allowing developers to reliably deploy containers for their applications. See this [link](https://aws.amazon.com/ecr/) for additional information on ECR. Currently, you have to use ECR to store container images for lambdas as it does not support other storage options such as Github, Dockerhub, etc.
+
+## Pros and Cons of using container for lambdas
+
+### Pros
+- Defining runtime environment in a container image gives developers more control over their environment compared to what they get with predefined runtimes and zipping dependencies. 
+- you can now package and deploy Lambda functions as container images of up to 10 GB in size compared to only 250MB in case of lambda deployment package size. 
+- preferable for data-heavy or dependency-heavy applications.
+- The Lambda service provides a variety of base image options with pre-installed runtimes that will be patched and maintained by AWS.
+- Once deployed, containerized lambdas have no additional cost compared to zipped lambdas except the cost of using ECR repository.
+
+### Cons
+- Container support requires your Lambda function code point to an ECR Repo URI which means that repo also has to be maintained.
+- need additional work to create the Dockerfile, tag and push the image to ECR repo.
+- Need additional work for deleting older images under the ECR repository.
+
+:::warning
+Currently NGAP does not support using ECR repository to store images which could bring possible risk and challenges to using containers for lambdas.
+:::
+
+## New configuration for lambda container
+There are a few configuration that needs to be added if the lambda is deployed from docker image.
```

Comment:
You may want to mention that you should use the AWS lambda base image specific to the language you are using for Lambda.

Reply:
added